netlify.toml

```diff
@@ -1,3 +1,11 @@
 [build]
 publish = "_site"
 command = "./build-website.sh"
+
+[[headers]]
```

Comment:
This does not appear to be working correctly from the preview. Trying to figure it out. cc @dschaller

Reply:
according to https://docs.netlify.com/routing/headers/#syntax-for-the-headers-file this should go into the "_headers" file. I'm not sure exactly where that file should go, or if that's supposed to be a suffix to some other filename.

Reply:
It can also live in the config. If you go the headers file route you’ll need to include that as part of the Jekyll config since _ prefixed files are ignored by default. I can dig more when I land

Reply:
Starting to wonder if the headers don't work in preview. I see x-robots-tag header being set on some pages on the site but no idea how that is happening.

Reply:
I just merged this to see how it behaves on a real deploy. The behavior I was seeing in preview makes no sense to me. If it behaves the same on real deploy I will just revert it.

Reply:
As far as I know, specifying headers in the `netlify.toml` and in a `_headers` file should produce equivalent behaviors.

Just for clarification: what's the expected behavior versus the current behavior? A handful of exploratory `curl`s suggests that the headers are working as expected.